src/Workspaces/Core/Portable/Workspace/Solution/ProjectId.cs

```diff
@@ -28,8 +28,12 @@ private ProjectId(string debugName)
             _debugName = debugName;
         }
 
-        internal ProjectId(Guid guid, string debugName)
+        public ProjectId(Guid guid, string debugName = null)
```

Comment:
@jasonmalinowski the create style method is create new id, this would be creating an existing ID.  Feel like it would have to be a different named method and at that point we're implementing constructors with named static methods.  Is that what we want here?